index.src.html

```diff
@@ -616,6 +623,15 @@ <h5 id="algorithm-create-cred" algorithm>`[[Create]]` internal method</h5>
         a user has just clicked "sign-in" for example, then they won't be surprised or confused to
         see a [=credential chooser=] if necessary.
 
+    :   <dfn>conditional</dfn>
+    ::  Discovered credentials are presented to the user in a non-modal dialog along with an
+        indication of the origin which is requesting credentials. If the user makes a gesture
+        outside of the dialog, it closes and the `get()` method returns to the caller empty-handed
```

Comment:
>  it closes and the `get()` method returns to the caller empty-handed

I would rather not do this, otherwise the RP would be able to tell the user had credentials and decided not to exercise them. Let's either not resolve the promise, or resolve after `timeout`.